pyproject.toml

```diff
@@ -13,10 +13,8 @@ classifiers = [
 ]
 description = "Lightweight bluesky-as-a-service wrapper application. Also usable as a library."
 dependencies = [
-    "bluesky>=1.13.1",
+    "bluesky[plotting]>=1.13.1",
```

Comment:
Does `plotting` get us transitive dependencies we need? If so then all good but worth a comment explaining, it's not intuitive that we'd need the plotting parts for a headless service.